cs/unittest/RunTests.tt

```diff
@@ -6,121 +6,102 @@
 <#@ import namespace="System.Text" #>
 <#@ import namespace="System.Text.RegularExpressions" #>
 <#@ import namespace="System.Collections.Generic" #>
+<#@ assembly name="Newtonsoft.Json" #>
```

Comment:
maybe do the expanded hintpath https://github.com/VowpalWabbit/vowpal_wabbit/blob/5954860bfdb57e722d13c068cb75478c372abedd/cs/unittest/cs_unittest.csproj#L113 ugly but might work

Reply:
its worth trying

Reply:
now cmake fails and the other one passes :|

Reply:
That's not great...